src/Components/Authentication/Desktop/ModalManager.tsx

```diff
@@ -9,28 +9,36 @@ import {
   ModalType,
 } from "Components/Authentication/Types"
 
-interface Props {
+export interface ModalManagerProps {
   submitUrls: { [P in ModalType]: string }
   csrf: string
   redirectUrl?: string
-  type?: ModalType
   handleSubmit?: (
     type: ModalType,
     values: InputValues,
     formikBag: FormikProps<InputValues>
   ) => void
 }
 
-interface State {
-  currentType: ModalType
+export interface ModalManagerState {
+  currentType?: ModalType
 }
 
-export class ModalManager extends Component<Props, State> {
+export interface ModalOptions {
```

Comment:
@xtina-starr We'll need to use the interface you wrote here. Just putting a placeholder for now